packages/labs/ssr/src/test/lib/render-result-readable_test.ts

```diff
@@ -40,6 +40,27 @@ test('RenderResultReadable collects strings and nested Promises of iterables', a
   assert.equal(s, 'abcd');
 });
 
+test('RenderResultReadable collects all iterables 2', async () => {
```

Comment:
Nit: Could the test name be updated to be more specific of what is being tested?